system-probe_x64/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM debian:bullseye
+
+ARG BCC_VERSION=v0.12.0
+
+RUN apt update && apt full-upgrade -y
+RUN apt install -y \
+        clang \
+        git \
+        go-dep \
+        golang \
+        libelf-dev \
+        linux-headers-amd64 \
+        llvm \
+        python3-distutils \
+        python3-invoke \
+        python-pip
+
+RUN apt install -y \
```

Comment:
We will have to disribute libbcc, right? We might want to build it with omnibus (using this image) instead of here, so it gets installed in the prefix where it's going to be used from. Or is there another approach you are going for?

Reply:
`libbcc` build has been moved to DataDog/datadog-agent.